virt/virt-2-4-release-notes.adoc

```diff
@@ -12,10 +12,20 @@ include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+2]
 == New and changed features
 //Placeholder for new content.
 
-//CNV-4686 - Container-native Virtualization x" in the SVVP catalog listing
+//CNV-4686 - Container-native Virtualization x" in the SVVP catalog listing - Jira story was obsoleted.
 
 //CNV-4687 - Ensure platform passes Windows Server Virtualization Program with RHCOS workers.
 
+* {VirtProductName} is certified in Microsoft's Windows Server Virtualization Validation Program (SVVP) to run Windows Server workloads.
+
++
+The SVVP Certification applies to:
+
+** Red Hat Enterprise Linux CoreOS 8 workers. In the Microsoft SVVP Catalog, they are named Red Hat {product-title} 4 on RHEL CoreOS 8_.
```

Comment:
s/Red Hat {product-title}/_Red Hat {product-title}
for the italics markup

Reply:
Italics markup added.

Reply:
Italics markup added.